docs: VS code debugging config & MRE (#2440)
netlify/cli: docs/vscode.md

---

> title: Usage with Visual Studio Code

Reviewer: This may need a more specific title. Would this work?

Another option would be `Run and debug with Visual Studio Code`, per the comment below

```suggestion
title: Debugging with Visual Studio Code
```

---

> # Usage with Visual Studio Code

Reviewer: ```suggestion
# Run and debug with Visual Studio Code
```

This guide will walk you through configuring VScode to run and debug `netlify dev`.

> That allows debugging your site and functions all in the same IDE.

Reviewer: Small suggested wording tweak: 

```suggestion
This allows debugging your site and serverless functions all in the same IDE.
```
